packages/mui-icons-material/src/utils/SvgIcon.js

```diff
@@ -0,0 +1,234 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import {
+  unstable_capitalize as capitalize,
+  unstable_composeClasses as composeClasses,
+} from '@mui/utils';
+import { createStyled, shouldForwardProp, useThemeProps as systemUseThemeProps } from '@mui/system';
+import { defaultTheme } from '@mui/md-theme';
```

Comment:
This part increases the unnecessary bundle size for Joy UI and MD3.

Reply:
Yeah, it's unfortunate necessary thing for making sure we don't introduce breaking changes. Jun, if you feel like you are ok with having https://mui.com/joy-ui/guides/using-icon-libraries/ till v6, I am happy to go with the other PR 👍